libraries/riot/meta/summary.md

```diff
@@ -0,0 +1,9 @@
+<h4 id="riot-handling-data">Handling data</h4>
+
+By default, Riot.js passes all data to Custom Elements as attributes. If any of the attributes is not a primitive Riot.js passes it down to the Custom Element as DOM node property <code>domNode.objectAttribute = anObjectAttribute</code>
```

Comment:
```suggestion
By default, Riot.js passes all primitive data (strings, numbers, booleans) to Custom Elements as attributes. It passes complex data (Objects, Arrays) to Custom Elements as properties.
```

Tried to rephrase this a bit to simplify it.

Reply:
Thank you it makes sense.